Makefile

```diff
@@ -106,6 +106,22 @@ helm-lint: $(HELM) ## Validate helm chart.
 	$(HELM) template deploy/helm/grafana-operator/ > /dev/null
 	$(HELM) lint deploy/helm/grafana-operator/
 
+.PHONY: hugo
+hugo: $(DART_SASS) $(HUGO) ## Prepare production build for hugo docs.
+	$(info $(M) running $@)
+	@echo -- Checking presence of dart-sass
+	@cd hugo && $(HUGO) env | grep dart-sass
+	@echo -- Building artifacts
+	@cd hugo && HUGO_ENVIRONMENT=production HUGO_ENV=production $(HUGO) --gc --minify
```

Comment:
Should `postCSS` be installed as well or leave that up to users?

```bash
make hugo
▶ running hugo
-- Checking presence of dart-sass
github.com/sass/dart-sass/protocol="3.1.0"
github.com/sass/dart-sass/compiler="1.86.0"
github.com/sass/dart-sass/implementation="1.86.0"
-- Building artifacts
Start building sites …
hugo v0.134.3-5fb333b9f5ff4ec018f9d78910177d0bec1befb0+extended linux/amd64 BuildDate=2024-09-19T14:28:20Z VendorInfo=gohugoio

Total in 1575 ms
Error: error building site: POSTCSS: failed to transform "/scss/main.css" (text/css). You need to install PostCSS. See https://gohugo.io/functions/css/postcss/: binary with name "postcss" not found using npx
make: *** [Makefile:115: hugo] Error 1
```

Reply:
I think I figured out how it can done:

We can create a file with a name that would include hash of npm lock file (only after all deps are installed), so we can run it only when deps change by leveraging Makefile's built-in checks (the same as for other binaries). I'll try it out on Monday or so :)